File: app/src/main/java/xyz/farshad/vocab/model/Course.java
package xyz.farshad.vocab.model;

import com.orm.SugarRecord;

/**
 * Created by farshad on 9/29/15.
 */
public class Course extends SugarRecord {

    String name;
    int wordCount;

    public Course(){

    }

    public Course(String name, int wordCount){
        this.name = name;
        this.wordCount= wordCount;
    }

    public String getName() {
        return name;
    }

    public int getWordCount() {
        return wordCount;
    }

}

File: app/src/main/java/xyz/farshad/vocab/model/Course.kt
package xyz.farshad.vocab.model

import com.orm.SugarRecord

/**
 * Created by farshad on 9/29/15.
 */
class Course : SugarRecord {

    var name: String
        internal set
    var wordCount: Int = 0
        internal set

    constructor() {

    }

    constructor(name: String, wordCount: Int) {
        this.name = name
        this.wordCount = wordCount
    }

}
